src/lib/components/Speakers.svelte

```diff
@@ -6,7 +6,7 @@
 <div class="flex flex-col gap-2 md:gap-8">
 	{#each speakers as speaker}
 		<a class="!text-secondary-500 !no-underline" href={`/speakers/${speaker?.slug}`}>
-			<div class="flex flex-col gap-8 bcu-card variant-ringed-secondary p-8">
+			<div class="flex flex-col gap-8 bcu-card variant-ringed-secondary p-8 bg-black">
```

Comment:
don't hardcode here, it doesn't look right on light mode.
Maybe it should go to a `variant-ghost-secondary`?

You can see more examples at https://www.blackcatui.com/docs/variants

Reply:
I tried variant-ghost-secondary and it didn't have enough contract in light mode. I switched secondary to primary and it looks good with both.